flutter_app/test/widget/change_port_protocol_widget_test.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:imacs/modules/mavlink_communication.dart';
+import 'package:imacs/widgets/change_port_protocol_widget.dart';
+
+void main() {
+  group('Change Port/Protocol Widget', () {
+    testWidgets(
```

Comment:
Add a which actually changes the params and checks (I know this functionality isn't done yet but it's good to have tests early on)